.circleci/config.yml

```diff
@@ -0,0 +1,97 @@
+version: 2
+
+# --------------------------------
+# common : base
+# --------------------------------
+
+base_node: &base_node
+  docker:
+  - image: circleci/node:8.10.0-browsers
```

Comment:
please update all yaml array indetations to be:
```
  docker:
    - image: circleci/node:8.10.0-browsers
```
instead of
```
  docker:
  - image: circleci/node:8.10.0-browsers
```

Reply:
@neilrussell6 this should be resolved now